dockertest/environment_unittests.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+# Pylint runs from a different directory, it's fine to import this way
+# pylint: disable=W0403
+
+import unittest
+
+class FakeCheckOutput(object):
+
+    exception = None
+    callcount = 0
+    command = None
+    name = None
+    version = '1.0'
+    release = 'a'
+    arch = '8088'
+    output_fmt = '%s#%s#%s#%s\n'
+    allnvras = tuple()
```

Comment:
Initializing a class attribute to a non-imutable is a no-no in python, I believe pylint will make quite a bit of noise about it.  It would be analogous to initializing a C pointer to some pre-determine address in a general program (not something specialized like a hardware driver or the kernel) instead of using ``malloc()`` or it's friends.  It causes lots of headaches, esp. in testing.  The best place for such things are in ``__new__()`` (for classes) and ``__init__()`` (for instances), sometimes both.

In this case, since there's really [only a single/special use for this](https://github.com/autotest/autotest-docker/pull/744/files#diff-185f8e9777f5988a94866cb9868e305bR93), maybe I should just move that, there.